source/common/network/dns_impl.cc

```diff
@@ -183,9 +184,13 @@ void DnsResolverImpl::PendingResolution::onAresGetAddrInfoCallback(int status, i
       // portFromTcpUrl().
       // TODO(chaoqin-li1123): remove try catch pattern here once we figure how to handle unexpected
       // exception in fuzz tests.
-      ENVOY_LOG_EVENT(debug, "cares_dns_resolution_complete",
-                      "dns resolution for {} completed with status {}", dns_name_,
-                      resolution_status);
+      ENVOY_LOG_EVENT(
```

Comment:
For c-ares due to how it's implemented, this is not going to show all addresses, right? Like it would only show v4 and return?